functions-and-scope.js

```diff
@@ -71,6 +124,19 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 9
+function highestScore(grades){
+
+    let highestScore = 0;
```

Comment:
Het is altijd beter om te beginnen met de eerste waarde van de array. Beetje flauw bij dit voorbeeld, maar stel dat je een array hebt van `[-2, -5, -1, -3]`. Als nu deze array aan jouw functie mee zou geven, dan komt daar `0` uit, terwijl dat cijfer helemaal niet in de array voorkomt.

```suggestion
    let highestScore = grades[0];
```